dist/index.js

```diff
@@ -6089,13 +6087,17 @@ const distributionSpecificAptDependencies = {
         "python-rosdep",
         // python required for sourcing setup.sh
         "python",
+        "libc++-dev",
+        "libc++abi-dev",
     ],
     focal: [
         // python-rosdep does not exist on Focal, so python3-rosdep is used.
         // The issue with ros-melodic-desktop-full is also non-applicable.
         "python3-rosdep",
         // python required for sourcing setup.sh
         "python",
+        "libc++-dev",
+        "libc++abi-dev",
     ],
     jammy: [
         // python-rosdep does not exist on Jammy, so python3-rosdep is used.
```

Comment:
Could you add a comment below the `python3-rosdep` line to mention that `libc++*` is intentionally not installed, and mention issue #506?

Also, please modify [the Typescript file](https://github.com/ros-tooling/setup-ros/blob/master/src/package_manager/apt.ts) and not the generated JS file directly. See the [development instructions](https://github.com/ros-tooling/setup-ros/blob/master/DEVELOPING.md#build-and-test). I can do this for you if you don't have Node/npm installed and don't want to install it.

Reply:
Done. Not sure if so many changes in .js file are expected after rebuilding, can you confirm if it's ok?

Reply:
That's okay, it happens from time to time.